### PR TITLE
OLS-906: Benchmark can generate histogram

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Benchmark can generate histogram

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)

## Related Tickets & Documents

- Related Issue #[OLS-906](https://issues.redhat.com//browse/OLS-906)

### CODE REVIEW

File: tests/config/invalid_config.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      max_entries: 1000
+  logging_config:
+    logging_level: INFO
+  default_provider: p1
+  default_model: m1
+  user_data_collection:
+    transcripts_disabled: true
+dev_config:
+  disable_auth: true
+  disable_tls: true
```
Comment (onmete) [inline tests/config/invalid_config.yaml:40]: Should it be here? What is invalid?

Comment (tisnik) [reply tests/config/invalid_config.yaml:40]: it is required by some benchmark. And `type` is invalid?

Comment (onmete) [reply tests/config/invalid_config.yaml:40]: so can you make a note on the line, to be visible on first sight why it is invalid?

Comment (tisnik) [reply tests/config/invalid_config.yaml:40]: done @onmete

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-e2e-tests-rosa / lightspeed-service"
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

**Pull request merged**